cmd/bitrot-streaming.go

```diff
@@ -84,9 +84,11 @@ func newStreamingBitrotWriter(disk StorageAPI, volume, filePath string, length i
 		totalFileSize := bitrotSumsTotalSize + length
 		err := disk.CreateFile(volume, filePath, totalFileSize, r)
 		if err != nil {
-			logger.LogIf(context.Background(), err)
-			r.CloseWithError(err)
+			reqInfo := (&logger.ReqInfo{}).AppendTags("storageDisk", disk.String())
+			ctx := logger.SetReqInfo(context.Background(), reqInfo)
+			logger.LogIf(ctx, err)
 		}
+		r.CloseWithError(err)
```

Comment:
Basically, I think we don't need to close the pipe reader. Closing the pipe writer is enough, it will send EOF to the pipe reader when pipeWriter.Close() is called. 

If I understood well, closing pipe reader means saying to the code sending data via the pipwriter, sorry I am not able to accept data anymore.

Reply:
> If I understood well, closing pipe reader means saying to the code sending data via the pipwriter, sorry I am not able to accept data anymore.

Yes, this is what we want. Basically when there is a network error, Write() on the pipe should get error (in erasure encode)

Reply:
> Yes, this is what we want. Basically, for ex., when there is a network error, Write() on the pipe should get error (in erasure encode)

oups yes, I see